### DISCUSSION TITLE
Strip OvenMediaEngine executable after RELEASE build

### DISCUSSION THREAD

**Discussion by vampirefrog** [OvenMediaLabs/OvenMediaEngine#1615] [Ideas]
Howdy!

On linux, the OvenMediaEngine executable turns out to be over 450MB in size after doing `make release`.

Please add a `strip OvenMediaEngine` command in the Makefile at the end of the `release` build.

**Comment by dimiden** [comment]
This is intentional, as there are cases where we need to obtain symbol information from the released binary. Therefore, during the `make release` step, we do not run `strip`, but instead, we strip the binary only during the `make install` process.
